stl/inc/format

```diff
@@ -2459,7 +2387,7 @@ struct _Invalid_format_kind {
 };
 
 _EXPORT_STD template <class _Ty>
-constexpr _Invalid_format_kind<_Ty> format_kind{};
+inline constexpr _Invalid_format_kind<_Ty> format_kind;

```

Comment:
this seems to be a stray change... Also I don't think we typically write this redundant inline

Reply:
We conventionally _do_ write this `inline`, because MSVC doesn't properly give variable templates external linkage without `inline` (or `extern`, with `/Zc:externConstexpr` which is implied by `/permissive-`).

Reply:
Demo:
```
C:\Users\cacarter\OneDrive - Microsoft\Desktop>type repro.cpp
#include <cassert>

template <class>
INLINE constexpr int meow = 42;

const int* f();

int main() {
    assert(&meow<void> == f());
}

C:\Users\cacarter\OneDrive - Microsoft\Desktop>type repro2.cpp
template <class>
INLINE constexpr int meow = 42;

const int* f() {
    return &meow<void>;
}

C:\Users\cacarter\OneDrive - Microsoft\Desktop>cl /nologo /std:c++17 /DINLINE="" repro.cpp repro2.cpp && repro && echo PASS || echo FAIL
repro.cpp
repro2.cpp
Generating Code...
Assertion failed: &meow<void> == f(), file repro.cpp, line 9
FAIL

C:\Users\cacarter\OneDrive - Microsoft\Desktop>cl /nologo /std:c++17 /DINLINE=inline repro.cpp repro2.cpp && repro && echo PASS || echo FAIL
repro.cpp
repro2.cpp
Generating Code...
PASS
```